mllib/src/main/scala/org/apache/spark/ml/clustering/LDA.scala

```diff
@@ -880,11 +880,13 @@ class LDA @Since("1.6.0") (
   }
 }
 
-
-private[clustering] object LDA extends DefaultParamsReadable[LDA] {
+@Since("2.0.0")
+object LDA extends DefaultParamsReadable[LDA] {
```

Comment:
For discussion: Should we bump up the version at L899 to 2.0.0? ```@Since``` is used for public function and ```LDA.load``` become public since 2.0.0 actually. cc @mengxr @jkbradley @MLnick